Makefile

```diff
@@ -3,12 +3,14 @@ include mk/common.mk
 ARCH ?= $(shell uname -m)
 PWD ?= $(shell pwd)
 
+SANITIZE = -fsanitize=thread
```

Comment:
Don't do that. Minimize the necessary changes.

Reply:
I added it accidentally. I've removed this change in the new revision.